modules/specialized-guides/pages/large-deployments/hub-api.adoc

```diff
@@ -4,11 +4,12 @@
 
 Make sure the ``hub-xmlrpc-api`` service is started:
 
+// CHECKIT!
 ----
-systemctl start hub-xmlrpc-api
+mgrctl exec -ti -- systemctl start hub-xmlrpc-api
```

Comment:
This does not work.

hub-xmlrpc-api is in separate container and is enabled by mgradm option --hubxmlrpc-replicas 1

Reply:
This probably means that there is no need anymore to mention this systemctl here.